Common/src/Common/Controller/Lva/AbstractBusinessDetailsController.php

```diff
@@ -85,11 +87,17 @@ public function indexAction()
             $data = Mapper::mapFromResult($orgData);
         }
 
+        $response = $this->handleQuery(ApplicationQuery::create(['id' => $this->getIdentifier()]));
+        $application = $response->getResult();
+
+        // Remove option to add subsidiary companies on PSV applications
+        $isLicenseApplicationPSV = $application['goodsOrPsv']['id'] === RefData::LICENCE_CATEGORY_PSV;
```

Comment:
One **really small** thing on variable names - we camel case them. It's in the coding standards "somewhere". We're not at a high enough coding standard level yet for the checks to cover it, but as we progress cleaning up the codebase, this will/would fail the checks.

So it'd need to be `$isLicenceApplicationPsv` in a few places here.

All LGTM otherwise 👍